authors/apps/authentication/migrations/0001_initial.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.1.2 on 2018-10-26 12:22
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('auth', '0009_alter_user_last_name_max_length'),
+    ]
+
+    operations = [
```

Comment:
We agreed that we were not supposed to push migrations to VCs, moving forward we should `.gitignore` them, but until then we can stay away from committing them.

Reply:
Hey @krmroland , for this particular instance we are going to be pushing the migrations.
Unfortunately, we can't have a hard and fast rule for it so in the mean time we shall be pushing migrations.
We can discuss this further during stand ups.

Reply:
Okay